frontend/src/index.tsx

```diff
@@ -35,6 +35,10 @@ const queryClient = new QueryClient({
       refetchOnWindowFocus: process.env.NODE_ENV === "production",
       queryFn: defaultQueryFn,
       staleTime: 60000,
+      networkMode: "always",
+    },
+    mutations: {
+      networkMode: "always",
```

Comment:
Added to enable running the project offline. If these flags are not set, react query will not even try to execute the fetch queries if you have no internet connection.